cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -89,11 +90,11 @@ if [[ "${#buckets[@]}" -gt 0 ]]; then
   timeout 30m gsutil -m rm -r "${buckets[@]}"
 fi
 
-FAILED="$(ls "${ARTIFACTS}"/deprovision/*/failure)"
+FAILED="$(find ${clusters} -name failure -printf '%H\n')"
 if [[ -n "${FAILED}" ]]; then
   echo "Deprovision failed on the following clusters:"
-  echo "${FAILED}"
+  sed 's,.*/,,' <<< $FAILED
```

Comment:
This isn't incorrect, but using the `basename` CLI makes this much easier to undestand ;)

Reply:
Changed to `xargs --max-args 1 basename`.